cmd/ocsp-updater/main.go

```diff
@@ -33,16 +33,30 @@ type Config struct {
 		// E1 -> 1, R3 -> 3, etc.
 		Issuers map[string]int
 
-		OldOCSPWindow    cmd.ConfigDuration
+		// OldOCSPWindow controls how frequently ocsp-updater signs a batch
+		// of responses.
+		OldOCSPWindow cmd.ConfigDuration
+		// OldOCSPBatchSize controls the maximum number of responses
+		// ocsp-updater will sign every OldOCSPWindow.
 		OldOCSPBatchSize int
 
-		OCSPMinTimeToExpiry          cmd.ConfigDuration
+		// The worst-case freshness of a response during normal operations.
+		// This is related to to OCSPMinTimeToExpiry in ocsp-updater's config,
+		// and both are related to the mandated refresh times in the BRs and
+		// root programs (minus a safety margin).
+		ExpectedFreshness cmd.ConfigDuration
```

Comment:
The reference to this config key needs to be updated on line 146, and I think renaming this config key in this way violates our deployment guidelines.

Reply:
Oops, I didn't mean to rename this field. I set it back to OCSPMinTimeToExpiry.